easybuild/easyconfigs/t/tbl2asn/tbl2asn-02272018-linux64.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CmdCp'
+
+name = 'tbl2asn'
+version = '02272018'
```

Comment:
@pescobar I assume this is supposed to represent a datestamp?
Why not use `20180227` instead, since that sorts properly w.r.t. older vs newer versions?